pwnlib/asm.py

```diff
@@ -576,7 +576,7 @@ def make_elf(data,
         _run(assembler + ['-o', step2, step1])
 
         linker_options = ['-z', 'execstack']
-        if vma:
+        if vma != None:
```

Comment:
Python suggests the use of `is not` with None.

```suggestion
        if vma is not None:
```